cmd/tink-worker/main.go

```diff
@@ -22,35 +23,36 @@ func main() {
 
 	// Here we retry failed connections to tink-server using a randomized interval between attempts.
 	// tink-worker is a daemon process, so we do not exit here.
-	var conn *grpc.ClientConn
+	var creds credentials.TransportCredentials
 	for {
-		creds, err := tink.ObtainServerCreds(flagEnvSettings.TinkServerURL)
+		fmt.Printf("Obtaining tink server creds from %s...\n", flagEnvSettings.TinkServerURL)
```

Comment:
We should be writing log messages to stderr. I realize I didn't bring it up last time, but going forward we should use `log` or whatever terrible log library we decide to live with.

Reply:
I've migrated these to use `fmt.Fprintf` to write to `os.Stderr`